workflow/scripts/predict.py

```diff
@@ -49,6 +49,12 @@ class formatter(
         default=0.022,
         help="Threshold on ABC Score (--score_column) to call a predicted positive",
     )
+    parser.add_argument(
+        "--default_accessibility_feature",
```

Comment:
I'm wondering if `default_accessibility_feature` is a good name here. The input is only either DHS or ATAC right? If so, we can call it `input_assay_type` or something similar and then make it so that the user has to input either DHS or ATAC: https://docs.python.org/3/library/argparse.html#choices

Reply:
It's not. I used it here to be consistent with the call_neighborhood rule. I think it can be called `accessibility_feature`.